extra/ldr-to-hdr.sh

```diff
@@ -0,0 +1,226 @@
+#!/bin/bash
+
+# Setting up our work paths and file names
+OUTPUT_FILE=${!#}
+INPUT_FILES=0
+TEMP_FILEPATH="/tmp"
+TEMP_FILENAME=""
+TEMP_FILENAME_ALT="" # Used to go back-and-fourth on compiling the image.
+
+# Required files needed
+CONFIG_CALFAC_PATH=""	# Calibration Factor
+CONFIG_VIG_PATH=""	# Vignetting
+CONFIG_ND_PATH=""	# Neutral Density Filter
+CONFIG_FISHEYE_PATH=""	# Fisheye Correction
+CONFIG_CAMRESP_PATH=""	# Camera Response
+CONFIG_FISHEYE_DIAM=""	# Fisheye Diameter
+CONFIG_CROP_X=""	# Crop the image starting at X
+CONFIG_CROP_Y=""	# Crop the image starting at Y
+CONFIG_VA_V=""		# View angle of the lens vertically
+CONFIG_VA_H=""		# View angle of the lens horizontally
+CONFIG_OUT_X=""		# Output width of the image
+CONFIG_OUT_Y=""		# Output height of the image
+
+# Radiance prints the image to STDOUT, so we can use two variable to go back and fourth on applying modifiers.
+IMAGE_VAR1=""
+IMAGE_VAR2=""
+
+# Print Help function
+printhelp() {
+	echo "LDR 2 HDR"
+	echo "Automating the creation of HDR images from LDR sets"
+	echo "Outputs to CWD, with the HDR name being the MD5SUM"
+	echo "Required arguments:"
+	echo "-h [path] | HDRGen binary path"
+	echo "-r [path] | Radiance package binary path"
+	echo "-i [path/file wildcard] | Input image path/wildcard"
+	echo "-z [file] | Calibration Factoring File"
+	echo "-x [file] | Vignetting Config File"
+	echo "-c [file] | Neutral Density Filter File"
+	echo "-v [file] | Fisheye Correction File"
+	echo "-b [file] | Camera Response File"
+	echo "-n [file] | Camera Settings File"
+}
+
+# Check to see if no args were given
+
+if [[ -z $1 ]]; then
+	printhelp
+	exit 0
+fi
+
+# Argument Parsing
+
+# -h| HDRGEN path
+# -r| Radiance path
+# -i| Input images, can be regex or directory too.
+# ARUGMENTS ARE DESIGNED TO BE NEXT TO EACH OTHER ON THE KEYBOARD FOR EASE OF USE.
+# -z| Calibration Factoring File
+# -x| Vignetting Config File
+# -c| Neautral Density Filter File
+# -v| Fisheye Correction File
+# -b| Camera Response File
+# -n| Camera Settings File
+
+while getopts "h:r:i:z:x:c:v:b:n:" arg; do
+  case $arg in
+    h|r) # Setting HDRGEN path or radiance path
+      export PATH+=":$OPTARG"
+      ;;&
```

Comment:
On Mac. For me, this `;;&` threw an error. I took out the `&` and it didn't give me an issue.

Reply:
Accidentally added that, as I thought it was needed for OR operator case statement. Looks like it really isn't needed.

Removed.